scripts/globals/bcnm.lua

```diff
@@ -570,6 +570,7 @@ function checkReqs(player, npc, bfid, registrant)
         [ 993] = function() return ( cop == mi.cop.THE_WARRIOR_S_PATH                                                                                                       ) end, -- PM7-5: The Warrior's Path
         [1024] = function() return ( cop == mi.cop.WHEN_ANGELS_FALL and copStat == 4                                                                                        ) end, -- PM8-3: When Angels Fall
         [1056] = function() return ( cop == mi.cop.DAWN and copStat == 2                                                                                                    ) end, -- PM8-4: Dawn
+		[1057] = function() return ( player:getQuestStatus(JEUNO, tpz.quest.id.jeuno.APOCALYPSE_NIGH) == QUEST_ACCEPTED and player:getCharVar('ApocalypseNigh') == 4        ) end, -- Apocalypse Nigh
```

Comment:
No tabs.  Set your editor to use 4 spaces for indentation.

I use TextPad, and for this project,
```
configure > preferences > document classes > default >
    [X] Strip trailing spaces from lines when saving
    tabulation >
        default tab spacing 4
        indent size 4
        [X] convert new spaces to tabs
        [X] convert existing tabs to spaces when saving files
```

Reply:
Grabbed textpad and changed to those settings.